odinmain/src/main/kotlin/me/odinmain/features/impl/floor7/TickTimers.kt

```diff
@@ -0,0 +1,106 @@
+    private val showPrefix: Boolean by BooleanSetting("Show Prefix", default = true, description = "Shows the prefix of the timers")
+
+    private val necronHud: HudElement by HudSetting("Necron Hud", 10f, 10f, 1f, true) {
+        if (it) {
+            mcTextAndWidth(formatTimer(35, 60, "§4Necron dropping in"), 1f, 1f, 2, Color.DARK_RED, shadow = true, center = false) * 2 + 2f to 16f
+        } else if (necronTime >= 0) {
+            mcTextAndWidth(formatTimer(necronTime.toInt(), 60, "§4Necron dropping in"), 1f, 1f, 2, Color.DARK_RED, shadow = true, center = false) * 2 + 2f to 16f
+        } else 0f to 0f
+    }
+
+    private var necronTime: Byte = -1
+
+    private val goldorHud: HudElement by HudSetting("Goldor Hud", 10f, 10f, 1f, true) {
+        if (it) {
+            mcTextAndWidth(formatTimer(35, 60, "§7Tick:"), 1f, 1f, 2, Color.DARK_RED, shadow = true ,center = false) * 2 + 2f to 16f
+        } else if ((goldorStartTime >= 0 && startTimer) || goldorTickTime >= 0) {
+            val (prefix: String, time: Int, max: Int) = if (goldorStartTime >= 0 && startTimer) Triple("§aStart:", goldorStartTime, 104) else Triple("§7Tick:", goldorTickTime, 60)
+            mcTextAndWidth(formatTimer(time, max, prefix), 1f, 1f, 2, Color.DARK_RED, shadow = true ,center = false) * 2 + 2f to 16f
+        } else 0f to 0f
+    }
+    private val startTimer: Boolean by BooleanSetting("Start timer", default = false, description = "Displays a timer counting down until devices/terms are able to be activated/completed.").withDependency { goldorHud.enabled }
+
+    private var goldorTickTime: Int = -1
+    private var goldorStartTime: Int = -1
+
+    private val stormHud: HudElement by HudSetting("Storm Pad Hud", 10f, 10f, 1f, true) {
+        if (it) {
+            mcTextAndWidth(formatTimer(15, 20, "§bPad:"), 1f, 1f, 2, Color.DARK_RED, shadow = true ,center = false) * 2 + 2f to 16f
+        } else if (padTickTime >= 0) {
+            mcTextAndWidth(formatTimer(padTickTime, 20, "§bPad:"), 1f, 1f, 2, Color.DARK_RED, shadow = true ,center = false) * 2 + 2f to 16f
+        } else 0f to 0f
+    }
+
+    private var padTickTime: Int = -1
+
+    init {
+        onMessage(Regex("\\[BOSS] Necron: I'm afraid, your journey ends now\\."), { enabled && necronHud.enabled }) { necronTime = 60 }
+
+        onMessage(Regex("\\[BOSS] Goldor: Who dares trespass into my domain\\?"), { enabled && goldorHud.enabled }) { goldorTickTime = 60 }
+        onMessage(Regex("The Core entrance is opening!"), { enabled && goldorHud.enabled }) {
+            goldorTickTime = -1
+            goldorStartTime = -1
+        }
+
+        onMessage(Regex("\\[BOSS] Storm: I should have known that I stood no chance\\.")) {
+            if (goldorHud.enabled) goldorStartTime = 104
+            if (stormHud.enabled) padTickTime = 20
+        }
+
+        onMessage(Regex("\\[BOSS] Storm: I should have known that I stood no chance\\."), { enabled && stormHud.enabled }) { padTickTime = -1 }
```

Comment:
What

Reply:
this just turns off the timer when storm is done

Reply:
oh wait